packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -34,11 +34,11 @@ The System Log API has one endpoint:
 
 This collection resource is backed by a [LogEvent object](#logevent-object) and associated [event types](#event-types).
 
-See [Examples](#examples) for ways you can use the System Log API. For common use cases see [Useful System Log Queries](https://support.okta.com/help/Documentation/Knowledge_Article/Useful-System-Log-Queries).
+See [Examples](#examples) for ways you can use the System Log API. See [Useful System Log Queries](https://support.okta.com/help/Documentation/Knowledge_Article/Useful-System-Log-Queries) for common use cases.
 
 ## LogEvent object
 
-Each LogEvent object describes a single logged action or "event" performed by a set of actors for a set of targets.
+Each LogEvent object describes a single logged action or "event" that is performed by a set of actors for a set of targets.
 
 ### Example LogEvent object
```

Comment:
I'm noticing a lot of our code content contains personal or non-example type of date, e.g, which we should clean up: I'd change "alternateId" to "admin@example.com" and the "displayName" to "John Doe" or something else (John Fung is a former employee)